vision/classification/classification_utils.go

```diff
@@ -0,0 +1,18 @@
+package classification
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+
+	"github.com/fogleman/gg"
+	"go.viam.com/rdk/rimage"
+)
+
+// Overlay returns a color image with the classification label and confidence score overlaid on
+// the original image.
+func Overlay(img image.Image, label string, confidenceScore float64) (image.Image, error) {
+	gimg := gg.NewContextForImage(img)
+	rimage.DrawString(gimg, fmt.Sprintf("%v: %v", label, confidenceScore), image.Point{30, 30}, color.NRGBA{255, 0, 0, 255}, 30)
```

Comment:
I think here you can change the float to a percentage, or at least reduce the number of significant digits. 
You can change `%v` to `%.4f` to limit the number of digits after the decimal place to 4. Or, you can do 
`fmt.Sprintf("%v: %.2f %%", label, confidenceScore*100.)` to write it as a percentage.